README.md

```diff
@@ -48,6 +48,18 @@ cats.data.NonEmptyList
 ### Refined
 Codecs for [refined](https://github.com/fthomas/refined) types are derived, so if you have `Encoder[A]`, then you have `Encoder[A Refined P]` (where `P` is a predicate) for free. The same for decoders, so having `Decoder[A]` in implicit scope automatically gives you `Decoder[A Refined P]`.
 
+To use calypso-refined in an existing sbt project, add the following dependencies to your build.sbt:
+```scala
+libraryDependencies += "ru.m2" %% "calypso-refined" % "<version>"
+```
+```scala
+import eu.timepit.refined.types.string.NonEmptyString
+import ru.m2.calypso.refined._
+import ru.m2.calypso.syntax._
+
+NonEmptyString("Text").asBson // BsonString{value='Text'}
```

Comment:
I might be missing something, but I'm not sure `NonEmptyString` has an apply signature to allow for this particular sintax 🤔 It doesn't seem to compile. Maybe something like the following would work slightly better instead?
```
    import eu.timepit.refined.collection.NonEmpty // or maybe import eu.timepit.refined.predicates.all._
    import eu.timepit.refined._
    import ru.m2.calypso.syntax._
    import ru.m2.calypso.refined._

    refineV[NonEmpty]("Text").asBson // BsonString{value='Text'}
```
Just a minor nitpick.